Mathlib/Computability/AkraBazzi/AkraBazzi.lean

```diff
@@ -464,7 +464,7 @@ lemma isEquivalent_smoothingFn_sub_self (i : α) :
             have h_log_pos : 0 < log n := Real.log_pos <| by aesop
             simp only [one_div]
             rw [inv_sub_inv (by have := hn' i; positivity) (by aesop)]
-      _ =ᶠ[atTop] (fun (n : ℕ) => (log n - log (b i) - log n) / ((log (b i) + log n) * log n))  := by
+      _ =ᶠ[atTop] (fun (n : ℕ) ↦ (log n - log (b i) - log n) / ((log (b i) + log n) * log n))  := by
```

Comment:
Here's another character saved for you :-)
```suggestion
      _ =ᶠ[atTop] (fun (n : ℕ) ↦ (log n - log (b i) - log n) / ((log (b i) + log n) * log n)) := by

Reply:
Yes, I had missed this when I merged the PR.  I already opened a PR with this (and a couple more) changes: #15107.